Projekt_3/Elections Scraper.py

```diff
@@ -0,0 +1,109 @@
+"""Project 3: Elections Scraper"""
+"""Choose territorial level from this URL: https://volby.cz/pls/ps2017nss/ps3?xjazyk=CZ """
+
+import os
+import csv
+import requests
+import bs4
+
+
+def main():
+    url, csv_name = input_url_name()
+    village_urls, v_ids = (get_villages_url(url))
+    for v_url in village_urls:
+        data = get_village_data(v_url, v_ids[village_urls.index(v_url)])
+        make_csv(csv_name, data)
+    print("=" * 50)
+    print(f"All done, {csv_name} has been created!!!")
+
+
+def input_url_name():
+    try:
+        url = input("Please insert URL adress: ")
+        test_url = requests.get(url)
+        test_url.raise_for_status()
+        while True:
+            csv_name = input("Please enter name of your csv file: ") + ".csv"
+            forbidden_char = ["\\", "/", ":", "*", "?", '"', "<", ">", "|"]
+            status = True
+            for char in forbidden_char:
+                if char in csv_name:
```

Comment:
This can be simpler:
```
forbidden_char = {"\\", "/", ":", "*", "?", '"', "<", ">", "|"}
if set(csv_name) & forbidden_char:
  ...
```
It uses set intersection.